docs/source/debugging.md

```diff
@@ -26,3 +26,31 @@ It's possible to go minimal and debug actor execution with remote debugger like
 3. In a separate console connect to the debugger via network utility of your choice. The default port is 4444.
 
 ```nc localhost 4444```
+
+
+### Initramfs
+
+One of the biggest debugging challenges is exploring something in initramfs stage, as currently there is no network
+connectivity (this might change soon though).
+
+1. (can be skipped if you already ended up with an emergency console)
+To get access to the emergency console right after leapp execution in initramfs stage has finished you should add an
+`rd.break=leapp-upgrade` argument to the kernel commandline. One way to do this is by changing the code of the
```

Comment:
Another way is to edit the kernel cmdline in Grub. This way it can be changed just before entering the upgrade initramfs.

Reply:
Awesome addition! Can you post a sample command? I'd love to add it.

Reply:
It actually isn't a command, but on the grub screen you can do `ctrl` + `e` to edit the kernel cmdline (actually the whole grub entry). And then `ctrl` + `x` to save and boot the entry.

Or maybe this can be done with the grubby command before reboot.